src/HIE/Bios/Cradle.hs

```diff
@@ -378,6 +393,9 @@ cabalCradle wdir mc =
     , cradleOptsProg   = CradleAction
         { actionName = Types.Cabal
         , runCradle = cabalAction wdir mc
+        , getGhcPath = optional $ fmap trim $
+            -- Need to pass -v0 otherwise we get "resolving dependencies..."
+            readProcess "cabal" ["v2-exec", "which", "-v0", "--", "ghc"] ""
```

Comment:
Does this work for completely uninitialized cabal projects?
Some tests would be nice

Reply:
Yeah, i think that line would fail in windows
```
PS D:\dev\ws\haskell\hls> cabal v2-exec which -v0 -- ghc
cabal.exe: The program 'which' is required but it could not be found.
```